script.js

```diff
@@ -0,0 +1,52 @@
+const formBtn = document.querySelector('#form-button');
+const booksList = document.querySelector('.books-list');
+
+let books = [];
+
+const booksAdd = (id, title, author) => {
+  const li = document.createElement('li');
+  li.innerHTML = `<h2>${title}</h2>
+    <h3>${author}</h3>
+    <hr>`;
+  const deleteBtn = document.createElement('button');
+  deleteBtn.textContent = 'Remove';
+  li.insertBefore(deleteBtn, li.lastElementChild);
+  booksList.appendChild(li);
+  deleteBtn.addEventListener('click', () => {
+    books = books.filter((books) => {
+      if (books.title !== title && books.author !== author) {
+        return true;
+      }
+      return false;
+    });
```

Comment:
[ Optional Changes ]

- [ ] This implementation works fine, but what if you could have extracted the remove functionality into a removeBook function outside of this booksAdd function or block of code and then use that removeBook function in here so your code looks professional and lean 👍🏽